README.rst

```diff
@@ -0,0 +1,59 @@
+# Princeton Geniza Project Research Partnership
+
+
+Python/Django web application for a new version of the `Princeton Geniza Project
+<https://cdh.princeton.edu/projects/princeton-geniza-project/>`_.
+
+Python 3.8 / Django 3.1 / Postgresql
+
+
+.. image:: https://github.com/Princeton-CDH/geniza/workflows/unit_tests/badge.svg
+    :target: https://github.com/Princeton-CDH/geniza/actions?query=workflow%3Aunit_tests
+    :alt: Unit Test status
+
+.. image:: https://codecov.io/gh/Princeton-CDH/geniza/branch/main/graph/badge.svg
+   :target: https://codecov.io/gh/Princeton-CDH/geniza
+   :alt: Code coverage
+
+.. image:: https://requires.io/github/Princeton-CDH/geniza/requirements.svg?branch=main
+     :target: https://requires.io/github/Princeton-CDH/geniza/requirements/?branch=main
+     :alt: Requirements Status
+
+.. image:: https://github.com/Princeton-CDH/geniza/workflows/dbdocs/badge.svg
+    :target: https://dbdocs.io/princetoncdh/geniza
+    :alt: dbdocs build
+
+
+Development instructions
+------------------------
+
+Initial setup and installation:
+
+- Recommended: create and activate a python 3.8 virtualenv
+
+- Install required python dependencies::
+
+    pip install -r requirements/dev.txt
+
+- Copy sample local settings and configure for your environment::
+
+	cp geniza/settings/local_settings.py.sample geniza/settings/local_settings.py
+
+Remember to add a ``SECRET_KEY`` setting!
+
+- Create a new database and update your database settings accordingly
+
+- Run database migrations
+
+    python manage.py migrate
+
```

Comment:
At some point we should add a reminder right after this to also run `cd geniza && django-admin compilemessages` which will compile the `.po` files so microcopy/translated content is available.

Reply:
You also need to run `django-admin makemessages` from the `geniza` dir if you create any _new_ translatable content, but I don't think that one needs to be in the README.

Reply:
Thanks for noting these — if not in the readme, where should they go? seems important to document somewhere

Reply:
imo the `compilemessages` one should go in the README, since it's required to get up and running. the `makemessages` one i view as analogous to `makemigrations`: not something we need to document explicitly since it occurs somewhat rarely, there's existing django documentation, and if you're in a position where you need it you probably already know. but open to changing that.

Reply:
@thatbudakguy I went ahead and added it to the readme for now, since it's not a regular part of our process yet like makemigrations is. We can always revise as we figure out what information is actually useful!